fraqbot/Local/newjob.py

```diff
@@ -41,27 +42,55 @@ def listening_for(self, message):
                 (text.startswith('!job') or
                 text.startswith('!newjob')))
 
+    def _strip_and_parse_terms(self, term):
+        term_split = term.split(',')
+        if len(term_split) > 2:
+            term_split[2] = re.sub(
+                '^at ',
+                '',
+                term_split[2].strip()
+            )
+        term_zero_stripped = term_split[0].strip().lower()
+        return [
+            (term_zero_stripped),
+            (
+                term_split[1].strip().lower() if
+                (len(term_split) > 1 and
+                    len(term_split[1].strip()) > 0)
+                else term_zero_stripped
+            ),
+            (
+                term_split[2].strip().lower() if
+                (len(term_split) > 2 and
+                    len(term_split[2].strip()) > 0)
+                else term_zero_stripped
+            ),
```

Comment:
This reads funny to me, probably because because the parens aren't necessary around each list entry (I think.)
This should also become cleaner (and maybe fit on one line) if you strip the terms while splitting (see comment above.)